### PR TITLE
Added DeliveryState to the Order model [ZG-1122]

### PR DESCRIPTION
**Pull request by AlirezaGorji**

### CODE REVIEW

**Review by odzhuryk — COMMENTED** [review]
Did you check tests in the project?

**Comment by AlirezaGorji** [comment]
All UnitTests passed.

File: FortnoxAPILibrary/Entities/Employee.cs [hunk 1]
```diff
@@ -139,7 +139,7 @@ public class Employee
         /// Hourly pay
         /// </summary>
         [XmlElement(ElementName = "HourlyPay")]
-        public double? HourlyPay { get; set; }
+        public object HourlyPay { get; set; }
```
[the remark this answers is not in the document]
Comment (AlirezaGorji) [reply FortnoxAPILibrary/Entities/Employee.cs:142]: It was "double?" but it gets a TypeCast error when tries to cast the received XML to employer.
The returned XML node for Hourly Pay is 
<HourlyPay />

[the remark this answers is not in the document]
Comment (AlirezaGorji) [reply FortnoxAPILibrary/Entities/Employee.cs:142]: &lt;HourlyPay /&gt;

[the remark this answers is not in the document]
Comment (suncastor) [reply FortnoxAPILibrary/Entities/Employee.cs]: It should be done this way 
image

**Review by suncastor — APPROVED** [review]

**Review by odzhuryk — APPROVED** [review]

**Pull request merged**